src/coreclr/vm/arm64/asmhelpers.S

```diff
@@ -337,6 +337,13 @@ NESTED_END OnHijackTripThread, _TEXT
 
 #endif // FEATURE_HIJACK
 
+// void* PacStripPtr(void *);
+.arch_extension pauth
```

Comment:
do you have machine that doesn't have PAC, where you can validate if this will not break?

Reply:
yup, I'll give it go.

Reply:
This was a good shout as `xpaci` doesn't fall in the nop/hint space. This leads to SIGILL on non-PAC machines. Now switched to using `xpaclri` that is nop on non-PAC machines. Confirmed this by running all tests on a non-PAC machine.

Reply:
From https://devblogs.microsoft.com/oldnewthing/20220819-00/?p=107020, I see that it points that to use `xpaclri`, we should check for feature support. I am wondering if it is possible that we get `SIGILL` on different machines where we didn't test this out like surface, etc. 

![image](https://github.com/user-attachments/assets/93c1ee18-c792-4dee-ac10-cb55bc103747)

Reply:
Sure, we can test it on another non-PAC machine. 
The `xpacd` and `xpaci` instructions don't fall into the nop/hint space but `xpaclri` does [1], so using xpaclri should be fine on non-pac machines..
![image](https://github.com/user-attachments/assets/21c5f4ec-e22f-4d49-848f-4cd87b18cade)
![image](https://github.com/user-attachments/assets/3efb98ae-1796-473c-9367-13d44d645d5c)

[1] https://docsmirror.github.io/A64/2024-03/xpac.html